### ISSUE TITLE
Add support for base64

### ISSUE THREAD

**Issue by renarsvilnis** [Jam3/preloader#4]
Hi,
I'd like to see base64 support added.

I'm currently using the library with Webpack and `url-loader` that inlines small filesize assets in base64 such as:

```javascript
loader.add('data:image/png;base64,iVBORw0KGgoAA...5CYII=');
```

Currently the library isn't able to handle such case and fails at `preloader/lib/util/getExtension.js`

Could the library add a check if given url string is base64 encoded before looking for a file extension and just ignore the url?

I'm willing to help you implement it, if you are interested.

-----
**EDIT:** made a pull request #5 that adds support

Once [`is-base64`](https://github.com/miguelmota/is-base64) merges my fix, my pull request can drop the util function `isBase64`in favour of `is-base64` dependency.

**Comment by njam3** [comment]
Fixed as of npm version `2.1.0` @renarsvilnis Thanks for all the hard work!

**Comment by renarsvilnis** [comment]
Your welcome 👍
